GVFS/GVFS.UnitTests/Common/Database/PlaceholdersTests.cs

```diff
@@ -0,0 +1,428 @@
+﻿using GVFS.Common.Database;
+using GVFS.Tests.Should;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace GVFS.UnitTests.Common.Database
+{
+    [TestFixture]
+    public class PlaceholdersTests

```

Comment:
This class goes a little mock crazy so I'm not sure the value this brings vs the maintenance cost of all the mocking it is doing.